tests/coord.test.js

```diff
@@ -51,4 +52,29 @@ test('validate Coord: default values', () => {
     expect(defCoord.validate()).toBe(false);
     expect(defXCoord.validate()).toBe(false);
     expect(defYCoord.validate()).toBe(false);
-})
\ No newline at end of file
+});
+
+test('verify toString output', () => {
+    let c = new Coord();
+
+    for (let i = 0; i < 100; i++) {
+        c.setLocation(i, i);
+        expect(c.toString()).toBe('(' + i + ', ' + i + ')');
+    }
+});
+
+test('comparison of Coord objects', () => {
+    let c1 = new Coord(1, 1);
+    let c2 = new Coord(1, 1);
+    let c3 = new Coord(2, 2);
+
+    expect(c1.is(c2)).toBe(true);
+    expect(c1.is(c3)).toBe(false);
+});
```

Comment:
Can you also check that `c1` doesn't equal `Coord(1,2)` or `Coord(2,1)`?

Reply:
I added those checks.